src/scss/_header.scss

```diff
@@ -2,69 +2,92 @@
 	width: 100%;
 	background: url("../images/tan-wood.jpg") no-repeat center;
 	background-size: cover;
+
+	@media(min-width: $bp-medium) {
+		padding-bottom: 1.75em; //28px
+	}
 }
 
 .nav {
 	text-align: center;
 	width: 100%;
 }
 
-.logo--lg {
+.site-title {
+	padding-top: 1em; //16px
 	margin: auto;
-	display: none;
-	@media(min-width: 980px) {
-		display: inline;
-		margin: 1.75em 0;
+
+	@media(min-width: $bp-medium) {
+		width: 40%;
+		position: absolute;
+		left: 30%;
 	}
 }
 
-.logo--sm {
-	margin: auto;
+.site-logo {
+	background-size: cover;
 	display: block;
-	padding: 1.75em 0 1.25em;
-	@media(min-width: 980px) {
-		display: none;
+	height: 2em; //32px
+	width: 1.75em; //28px
+	margin: 0 auto;
+	overflow: hidden;
+	text-indent: -9999em;
+
+	@media(min-width: $bp-medium) {
```

Comment:
I removed the break point variables from my scss, so I'm no longer naming any of my media queries.